synapse/models/infotech.py

```diff
@@ -303,6 +450,46 @@ def getBaseModels():
                     ('reg:int', {'ptype': 'it:dev:int', 'doc': 'The integer value that was deleted (parsed from :reg).', 'ro': 1}),
                     ('reg:bytes', {'ptype': 'file:bytes', 'doc': 'The binary data that was deleted (parsed from :reg).', 'ro': 1}),
                 )),
+                ('it:prod:soft', {}, (
+                    ('name', {'ptype': 'str:lwr', 'ro': 1, 'req': 1,
+                              'doc': 'Name of the software'}),
+                    ('desc', {'ptype': 'str:txt', 'doc': 'A description of the software'}),
+                    ('desc:short', {'ptype': 'str:lwr', 'doc': 'A short description of the software'}),
+                    ('author:org', {'ptype': 'ou:org', 'doc': 'Organization responsible for the software', }),
+                    ('author:acct', {'ptype': 'inet:web:acct', 'doc': 'Web user responsible for the software', }),
+                    ('author:person', {'ptype': 'ps:person', 'doc': 'Person responsible for the software', }),
+                    ('url', {'ptype': 'inet:url', 'doc': 'URL relevant for the software', }),
+                )),
+
+                ('it:prod:softver', {}, (
+                    ('software', {'ptype': 'it:prod:soft', 'req': 1, 'ro': 1,
+                                  'doc': 'Software associated with this version instance.'}),
+                    ('verstr', {'ptype': 'it:dev:str', 'req': 1, 'ro': 1, 'ex': '1.0.2'
+                                'Version string associated with this version instance.'}),
+                    ('verstr:norm', {'ptype': 'str:lwr', 'doc': 'Normalized version of the version string.'}),
+                    ('arch', {'ptype': 'it:dev:str', 'doc': 'Software architecture.'}),
+                    ('semver', {'ptype': 'it:semver', 'doc': 'System normalized semantic version number.', }),
+                    ('semver:major', {'ptype': 'int', 'doc': 'Version major number', }),
+                    ('semver:minor', {'ptype': 'int', 'doc': 'Version minor number', }),
+                    ('semver:patch', {'ptype': 'int', 'doc': 'Version patch number', }),
+                    ('semver:prerelease', {'ptype': 'str:txt', 'doc': 'Semver prerelease string.', }),
```

Comment:
maybe we could tighten this name down to pre?

Reply:
that a quick change

Reply:
done